auto-complete-config.el

```diff
@@ -264,14 +264,16 @@
 
 ;; css
 
-;; Copied from company-css.el
+;; Property list borrowed from Company, source:
+;; https://github.com/company-mode/company-mode/blob/master/company-css.el
```

Comment:
BTW, we're in the clear with this, right? AC and company are both GPL3, so I'd expect this inclusion to be OK - is my intuition correct?

Reply:
Yes, this should be fine.